src/previewer.ts

```diff
@@ -111,8 +112,14 @@ class Previewer implements vscode.TextDocumentContentProvider {
         const previewFileType = config.previewFileType;
         const previewMimeType = previewFileType === 'png' ? 'png' : "svg+xml";
 
-        let task = exporter.exportToBuffer(diagram, previewFileType);
-        this.process = task.process;
+        let task;
+        if (config.previewFromUrlServer) {
+            task = httpExporter.exportToBuffer(diagram, previewFileType);
+        } else {
+            task = exporter.exportToBuffer(diagram, previewFileType);   
```

Comment:
In local export, this.process should be recorded for later processing of prevent from starting to many processes.